engines/cge2/detection.cpp

```diff
@@ -101,6 +103,28 @@ static const ADExtraGuiOptionsMap optionsList[] = {
 			}
 		},
 
+#ifdef USE_TTS
+	{
+		GAMEOPTION_TTS_THINGS,
+		{
+			_s("Enable Text to Speech for Objects and Options"),
+			_s("Use TTS to read the descriptions (if TTS is available)"),
+			"tts_enabled",
+			false
+		}
+	},
+
+	{
+		GAMEOPTION_TTS_SPEECH,
+		{
+			_s("Enable Text to Speech for Subtitles and Override Original Voice Overs"),
+			_s("Use TTS to read the descriptions (if TTS is available)"),
+			"tts_enabled_new",
```

Comment:
I'm not sure about the name of the config variable. it might be used in other engines, too - and should imo get a better speaking name.

Also I'm no native speaker, but the upper case writing of the description doesn't look correct for me. Is there a reason behind writing it like this? Like highlighting words with special meanings in the game?

Reply:
The upper case is there just because it was also there in the other engines I worked on, just thinking about uniforming the format.